packages/appsync-emulator-serverless/lambdaRunnerPython.js

```diff
@@ -0,0 +1,69 @@
+const log = require('logdown')('appsync-emulator:lambdaRunner');
+
+const parseErrorStack = error =>
+  error.stack
+    .replace(/at /g, '')
+    .split('\n    ')
+    .slice(1);
+
+const sendOutput = output => {
+  process.send({ type: 'success', output }, process.exit);
+};
+const sendErr = err => {
+  let error;
+  if (err instanceof Error) {
+    error = {
+      stackTrace: parseErrorStack(err),
+      errorType: err.constructor.name,
+      errorMessage: err.message,
+    };
+  } else {
+    error = err;
+  }
+  process.send({ type: 'error', error }, process.exit);
+};
+
+process.once(
+  'message',
+  async ({ serverlessDirectory, handlerPath, handlerMethod, payload}) => {
+    try {
+        log.info('load', module);
+   
+        var spawn = require('child_process').spawn          
+        var args = ["invoke", "local", "-f", handlerMethod]
+
+        var sls = spawn("sls", 
```

Comment:
curious why you choose to have serverless invoke python instead of invoking python.

Reply:
I choose serverless to invoke python because several reasons:

* it creates the context needed to be passed to the lambda
* it manage to invoke the lambda with all information needed
* it returns to the stdout the result of the lambda
  - when i try to do it with `python` call, it force me to use a print function to print directly the result returned by the handler function
something like: 
`python3 -c "import example; import sys; print(handler.example(payload, context), file=sys.stdout)"`